pkg/apis/pipeline/v1/result_validation.go

```diff
@@ -35,10 +35,10 @@ func (tr TaskResult) Validate(ctx context.Context) (errs *apis.FieldError) {
 	}
 
 	switch {
-	// Object are alpha features
+	// Object results is alpha feature - check if the feature flag is set to "beta" or "alpha"
```

Comment:
Object results, after this PR, will be a beta feature, right?

Reply:
> It looks like the PR will need a rebase though

Never mind, tide changed its mind...

Reply:
Thanks! I realized that I fixed it locally but forgot to push 😢